docs/RUNNING-LOCALLY.md

```diff
@@ -20,7 +20,7 @@ Prerequisites
 - [Rimraf](https://github.com/isaacs/rimraf)
 - [Asar](https://github.com/electron/asar)
 - [NSIS v2.51](http://nsis.sourceforge.net/Main_Page) (v3.x won't work)
-- [Visual Studio Community 2013](https://www.visualstudio.com/en-us/news/vs2013-community-vs.aspx)
+- [Visual Studio Community 2013 (free)](https://www.microsoft.com/en-us/download/details.aspx?id=48129)
```

Comment:
Never mind, you're right; I really should learn to not review things at five in the morning... I've come to this realization many times before but for some reason keep doing it 😒